docs/infra/environments.md

```diff
@@ -1,9 +1,71 @@
 # Hosted environments
 
-TBC
+## List of environments in use and the branches to which they are tied
 
-- List of environments in use and the branches to which they are tied
-- How deployments are triggered (GitHub Actions?)
-- General advice: Trust the pipeline and deploy small changes regularly
-- How to access a shell
-- How to pull data (link to developer-guide/fetching-production-data.md)
+The GitHub branch and Hosted environments have the same name:
+
+- `main`: for Live/Production
+- `develop` : for Development
+
+## How deployments are triggered
+
+- Deployments are manual at this time.
+
+### Pre-requisites
+
+- Requires user access on the hosted environment for deployments. Check with management if you need one.
+- Follow Branching Model and Naming branches [project-conventions] (<https://nationalarchives.github.io/ds-wagtail/developer-guide/project-conventions/>)
+- Check if it is Ok to deploy see guidance on deployment [.github\PULL_REQUEST_TEMPLATE.md](https://github.com/nationalarchives/ds-wagtail/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)
+- Test on local before deploy to environment
+
+### main
+
+#### Set environment variables from the UI console for `main` environment
```

Comment:
If these are steps, it might be clearer to number these headings.

Reply:
Thanks, Done https://github.com/nationalarchives/ds-wagtail/pull/682/commits/04930722dfa5e3642430f999d51f2f6215422b09